andela/bp-esa-backend
# #161545388 Migrate swagger documentation to postman

#### What does this PR do?
Migrates swagger documentation to Postman.

#### Description of Task to be completed?
 - Create postman collections.
 - Publish the said collections
 - Add 'Run in Postman' button on README

#### How should this be manually tested?
1. Git clone the repository.
2. Checkout to `ch-swagger-to-postman-161545388`.
3. Install necessary dependencies.
  `npm install` _(or the `yarn` equivalent)_.
4. Ensure the `.env` file is updated with the required variables.
5. Start the server `npm run start:dev` _(or the `yarn` equivalent)_.
6. On the README, click the `Run in Postman` button and it will automatically download and import the collections.
7. You can test the API endpoints and create a mock server on the imported collections.

#### Any background context you want to provide?
You should already have installed Postman.
You _may_ need a Postman account.
The collections' environment  variable `baseUrl` has been set to `http://localhost:8000`, you can change this to whatever host and port that the application has started on.

#### What are the relevant pivotal tracker stories?
* [#161545388](https://www.pivotaltracker.com/story/show/161545388)

#### Postman Documentation
Click this [link](https://documenter.getpostman.com/view/3235336/S11HteM1).

#### Screenshots (If applicable)
![image](https://user-images.githubusercontent.com/41262970/52480693-7dbf1980-2bbd-11e9-927e-62eaa180c79e.png)
